src/main/java/io/antmedia/streamsource/StreamFetcher.java

```diff
@@ -320,20 +333,23 @@ public void run() {
 					logger.info("Broadcast with streamId:{} should be deleted before its thread is started", streamId);
 					return;
 				}
-				else if (AntMediaApplicationAdapter.isStreaming(broadcast.getStatus())) {
+				else if (AntMediaApplicationAdapter.isStreaming(broadcast.getStatus()) && !broadcast.getCategory().equals("rtmp_origin_pull")) {
```

Comment:
Instead of doing the things with `if` statements, I suggest you two options because putting `category` to a value and checking it will likely cause other problems.

1. Create a subclass of StreamFetcher that just does its job. I mean pull the stream from the origin silently. 
2. Create a mode in the StreamFetcher that just pull the stream from the origin silently.